examples/virtual_staining/vcp_tutorials/quick_start.py

```diff
@@ -0,0 +1,333 @@
+
+# Run prediction
+trainer.predict(model=vs_cyto_2d, datamodule=data_module, return_predictions=False)
+
+# %% [markdown]
+"""
+# Model Outputs
+
+The model outputs are also stored in an OME-Zarr store.
+It can be visualized in an image viewer such as [napari](https://napari.org/).
+Below we show a snapshot in the notebook.
+"""
+
+# %%
+# Read images from Zarr stores
+# Choose the ROI for better visualization
+y_slice = slice(0, 512)
+x_slice = slice(0, 512)
+
+# Open the prediction store and get the 2D images from 5D arrays (t,c,z,y,x)
+with open_ome_zarr(output_path / fov) as vs_store:
+    vs_nucleus = vs_store[0][0, 0, 0, y_slice, x_slice]
+    vs_membrane = vs_store[0][0, 1, 0, y_slice, x_slice]
+
+# Open the experimental fluorescence dataset
+with open_ome_zarr(input_data_path / fov) as fluor_store:
+    fluor_nucleus = fluor_store[0][0, 1, 0, y_slice, x_slice]
+    fluor_membrane = fluor_store[0][0, 2, 0, y_slice, x_slice]
+
+# %%
+# Plot
+import matplotlib.pyplot as plt
+import numpy as np
+from cmap import Colormap
+from skimage.exposure import rescale_intensity
+
+
+def render_rgb(image: np.ndarray, colormap: Colormap):
+    image = rescale_intensity(image, out_range=(0, 1))
+    image = colormap(image)
+    return image
+
+
+# Render the images as RGB in false colors
+vs_nucleus_rgb = render_rgb(vs_nucleus, Colormap("bop_blue"))
+vs_membrane_rgb = render_rgb(vs_membrane, Colormap("bop_orange"))
+merged_vs = (vs_nucleus_rgb + vs_membrane_rgb).clip(0, 1)
+
+fluor_nucleus_rgb = render_rgb(fluor_nucleus, Colormap("green"))
+fluor_membrane_rgb = render_rgb(fluor_membrane, Colormap("magenta"))
+merged_fluor = (fluor_nucleus_rgb + fluor_membrane_rgb).clip(0, 1)
+
+# Plot
+# Show the individual channels and then fused in a grid
+fig, ax = plt.subplots(2, 3, figsize=(15, 10))
+
+# Virtual staining plots
+ax[0, 0].imshow(vs_nucleus_rgb)
+ax[0, 0].set_title("VS Nuclei")
+ax[0, 1].imshow(vs_membrane_rgb)
+ax[0, 1].set_title("VS Membrane")
+ax[0, 2].imshow(merged_vs)
+ax[0, 2].set_title("VS Nuclei+Membrane")
+
+# Experimental fluorescence plots
+ax[1, 0].imshow(fluor_nucleus_rgb)
+ax[1, 0].set_title("Experimental Fluorescence Nuclei")
+ax[1, 1].imshow(fluor_membrane_rgb)
+ax[1, 1].set_title("Experimental Fluorescence Membrane")
+ax[1, 2].imshow(merged_fluor)
+ax[1, 2].set_title("Experimental Fluorescence Nuclei+Membrane")
+
+# turnoff axis
+for a in ax.flatten():
+    a.axis("off")
+plt.tight_layout()
+plt.show()
+
+# %% [markdown]
+"""
+## Responsible Use
+
+We are committed to advancing the responsible development and use of artificial intelligence.
+Please follow our <Link text="Acceptable Use Policy" url="/acceptable-use-policy" />  when engaging with our services.
+
+Should you have any security or privacy issues or questions related to the services,
+please reach out to our team at <Email to="security@chanzuckerberg.com"/> or <Email to="privacy@chanzuckerberg.com" /> respectively.
```

Comment:
Sorry, I think my previous suggestion misled you. We can make the links appear as so:

```suggestion
Please follow our [Acceptable Use Policy](https://virtualcellmodels.cziscience.com/acceptable-use-policy) when engaging with our services.

Should you have any security or privacy issues or questions related to the services,
please reach out to our team at [security@chanzuckerberg.com](mailto:security@chanzuckerberg.com) or [privacy@chanzuckerberg.com](mailto:privacy@chanzuckerberg.com) respectively.
```